sf-docker/jetson-docker-compose.yml

```diff
@@ -170,12 +177,26 @@ services:
       - RabbitMQ__Password
       - RabbitMQ__Port
       - AppSettings__Log-RollingFile-Enabled
-      - AppSettings__Log_RabbitMq_Enabled
     volumes:
       - "./iengine.lic:/etc/innovatrics/iengine.lic"
     runtime: nvidia
 
-  extractor:
+  pedestrian-detector-gpu:
+    image: ${REGISTRY}sf-jetson-pedestrian-detector:${JETSON_VERSION}
+    container_name: SFPedestrianDetectGpu
+    command: --gpu=true
```

Comment:
I would remove this legacy console switch for GPU from all places and use Gpu__GpuEnabled from now on.
It is confusing + we are setting both env and console args here, which is also not best approach.